jest.config.js

```diff
@@ -7,4 +7,6 @@ module.exports = {
   collectCoverage: false,
   moduleFileExtensions: ['ts', 'js', 'json'],
   forceExit: true,
+  // `dotnet` commands on fresh machines requires restoration of modules which can take a while to resolve.
+  testTimeout: 60000,
```

Comment:
Since I prefer to run tests in parallel if possible, it's not predictable what test will be the one to take the long time to run and warm up the cache. So applied this globally.